### ISSUE TITLE
Add windows x64 compatibility

### ISSUE THREAD

**Issue by yckart** [teragonaudio/KickMaker#8]

**Comment by nikreiman** [comment]
Yes, this should definitely happen. :) Edited your issue title slightly, because actually making the plugin 64-bit compatible should happen on all platforms.

**Comment by nikreiman** [comment]
Requires #5 to be completed first.
